dojo/urls.py

```diff
@@ -166,7 +166,7 @@
     url(r'^api/v2/api-token-auth/', tokenviews.obtain_auth_token),
     url(r'^api/v2/doc/', schema_view, name="api_v2_schema"),
     url(r'^robots.txt', lambda x: HttpResponse("User-Agent: *\nDisallow: /", content_type="text/plain"), name="robots_file"),
-
+    url(r'^auth/', include('social_django.urls', namespace='social')),
```

Comment:
This produces warnings in the Initializer container. It is also redundant as it is already supplied in the dojo/users/urls.py.

Reply:
Agree, should be removed then.

Reply:
Agreed, will remove it